packages/aws-cdk-lib/aws-ecs/lib/log-drivers/aws-log-driver.ts

```diff
@@ -116,6 +131,11 @@ export class AwsLogDriver extends LogDriver {
       retention: this.props.logRetention || Infinity,
     });
 
+    var maxBufferSize = '';
```

Comment:
This needs to be `undefined` in the event that `props.maxBufferSize` isn't provided. Otherwise, `removeEmpty` on the log driver options won't remove `max-buffer-size`. The build failure is coming from other integ tests being impacted since with `maxBufferSize` being an empty string it'll be set on everything using this even when `props.maxBufferSize`. Isn't provided. Change this to:

`let maxBufferSize: string | undefined = undefined;`